### PR TITLE
docs(headless): edited recommendation documentation

### PR DESCRIPTION
**Pull request by jgeddes-coveo**

### CODE REVIEW

**Review by olamothe — APPROVED** [review]

File: packages/headless/src/controllers/recommendation/headless-recommendation.ts [hunk 1]
```diff
@@ -30,9 +30,10 @@ export interface RecommendationListProps {
 }
 
 /**
- * `Recommendation` controller allows to retrieve information about the current recommendations by the search API, if any.
+ * The `RecommendationList` controller allows users to retrieve information about the current recommendations by the search API, if there are any.
```
Comment (kaitlynkatz) [inline packages/headless/src/controllers/recommendation/headless-recommendation.ts:33]: Being nitpicky, but I wouldn't call the people using this controller "users"- it's the developers creating an interface who are going to be using headless controllers. I'm not sure if we can use "you" in reference docs; if we can, then I'd change users to you, and if not, I'd rephrase it as "...controller retrieves information..."

**Review by kaitlynkatz — APPROVED** [review]

**Review by ThibodeauJF — APPROVED** [review]
